src/System.EventLog/ref/System.EventLog.csproj

```diff
@@ -0,0 +1,13 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
+<Project ToolsVersion="14.0" xmlns="http://schemas.microsoft.com/developer/msbuild/2003" DefaultTargets="Build">
+  <Import Project="$([MSBuild]::GetDirectoryNameOfFileAbove($(MSBuildThisFileDirectory), dir.props))\dir.props" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Debug|AnyCPU'" />
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)' == 'netcoreapp-Release|AnyCPU'" />
+  <ItemGroup>
+    <Compile Include="System.EventLog.cs" />
+  </ItemGroup>
+  <ItemGroup>
+    <ProjectReference Include="..\..\System.Security.Permissions\ref\System.Security.Permissions.csproj" />
+  </ItemGroup>
```

Comment:
You also need this to fix the netfx build
```xml
  <ItemGroup Condition="'$(TargetGroup)' == 'netfx'">
    <Reference Include="mscorlib" />
     <Reference Include="System" />
  </ItemGroup>
```

Build locally with `build -framework:netfx` and you will get these errors. Then you can verify you don't need other references also. If you still get errors, figure out the desktop assembly they're in (like mscorlib, System) and add that also.

Reply:
I did this